docs/quickstart-packet.md

```diff
@@ -0,0 +1,269 @@
+INFO[14:20:56 EEST] Deploying PKI…
+INFO[14:20:56 EEST] Uploading files…                              node=147.75.81.119
+INFO[14:20:56 EEST] Uploading files…                              node=147.75.84.57
+INFO[14:21:01 EEST] Configuring certs and etcd on consecutive controller…
+INFO[14:21:01 EEST] Ensuring Certificates…                        node=147.75.81.119
+INFO[14:21:01 EEST] Ensuring Certificates…                        node=147.75.84.57
+INFO[14:21:11 EEST] Initializing Kubernetes on leader…
+INFO[14:21:11 EEST] Running kubeadm…                              node=147.75.80.241
+INFO[14:22:29 EEST] Joining controlplane node…
+INFO[14:22:29 EEST] Waiting 30s to ensure main control plane components are up…  node=147.75.84.57
+INFO[14:24:22 EEST] Waiting 30s to ensure main control plane components are up…  node=147.75.81.119
+INFO[14:26:21 EEST] Copying Kubeconfig to home directory…         node=147.75.81.119
+INFO[14:26:21 EEST] Copying Kubeconfig to home directory…         node=147.75.84.57
+INFO[14:26:21 EEST] Copying Kubeconfig to home directory…         node=147.75.80.241
+INFO[14:26:22 EEST] Building Kubernetes clientset…
+INFO[14:26:26 EEST] Creating credentials secret…
+INFO[14:26:26 EEST] Ensure external CCM is up to date
+INFO[14:26:27 EEST] Patching coreDNS with uninitialized toleration…
+INFO[14:26:27 EEST] Applying canal CNI plugin…
+INFO[14:26:31 EEST] Installing machine-controller…
+INFO[14:26:35 EEST] Installing machine-controller webhooks…
+INFO[14:26:37 EEST] Waiting for machine-controller to come up…
+INFO[14:27:17 EEST] Creating worker machines…
+```
+
+KubeOne automatically downloads the Kubeconfig file for the cluster. It's named
+as **\<cluster_name>-kubeconfig**, where **\<cluster_name>** is the name from
+your configuration. You can use it with kubectl such as
+
+```bash
+kubectl --kubeconfig=<cluster_name>-kubeconfig
+```
+
+or export the `KUBECONFIG` variable environment variable:
+```bash
+export KUBECONFIG=$PWD/<cluster_name>-kubeconfig
+```
+
+## Scaling Worker Nodes
+
+As worker nodes are managed by MachineController, they can be scaled up and down
+(including 0) using kubernetes scale API.
+
+```bash
+kubectl --namespace kube-system scale machinedeployment/pool1-deployment --replicas=3
+```
+
+## Deleting The Cluster
+
+Before deleting a cluster you should clean up all MachineDeployments, so all
+worker nodes are deleted. You can do it with the `kubeone reset` command:
+
+```bash
+kubeone reset config.yaml --tfjson tf.json
+```
+
+This command will wait for all worker nodes to be gone. Once it's done you can
+proceed and destroy the DigitalOcean infrastructure using Terraform:
+
+```bash
+terraform destroy
+```
+
+You'll be asked to enter `yes` to confirm your intention to destroy the cluster.
+
+Congratulations! You're now running Kubernetes 1.14.1 HA cluster with three
+control plane nodes and worker nodes. If you want to learn more about KubeOne
```

Comment:
`and one worker node`.

Reply:
That sounds bad and didn't want to focus attention on that. Also, docs on how to scale worker nodes are provided above.

Reply:
But it sounds better than this and grammatically more correct IMO. It's okay to mention it's one because that's was the plan all the way along.